src/java/fr/paris/lutece/plugins/forms/web/FormXPage.java

```diff
@@ -895,6 +881,8 @@ public synchronized XPage doSaveStep( HttpServletRequest request ) throws SiteMe
             // why are we here as we didn't try to save any backup ? So instead of throwing the error, we redirect.
             AppLogService.error("FormXPage l 897 : " + MESSAGE_ERROR_TOKEN );
             _currentStep = StepHome.findByPrimaryKey(Integer.parseInt(request.getParameter(FormsConstants.PARAMETER_ID_STEP)));
+            List<String> errorList = new ArrayList<>( );
+            _currentStep = FormsResponseUtils.getNextStep( _currentStep.getId( ), errorList, _formResponseManager );
```

Comment:
Are you sure that there won't be any regressions?

Reply:
This is already in the dev branch. We haven't seen any regression